_health-care/_js/store/calculated.js

```diff
@@ -0,0 +1,7 @@
+// Functions that produce calculated values given a state.
+//
+
+export function neverMarried(state) {
+  return state.nameAndGeneralInformation.maritalStatus === '' ||
```

Comment:
Yah...too many changes already in this PR.  Would rather avoid mingling in intentional business logic changes too.